package.json

```
@@ -2,18 +2,22 @@
   "name": "solc",
   "version": "0.8.10",
   "description": "Solidity compiler",
-  "main": "index.js",
+  "main": "dist/index.js",
   "bin": {
-    "solcjs": "solcjs"
+    "solcjs": "dist/solc.js"
   },
   "scripts": {
-    "lint": "node ./node_modules/semistandard/bin/cmd.js",
-    "updateBinary": "node downloadCurrentVersion.js && node verifyVersion.js",
-    "prepublishOnly": "npm run updateBinary",
-    "pretest": "npm run lint",
-    "test": "tape ./test/index.js",
-    "coverage": "node ./node_modules/nyc/bin/nyc.js --reporter=lcov --reporter=text-summary ./node_modules/tape/bin/tape ./test/index.js",
-    "coveralls": "npm run coverage && node ./node_modules/coveralls/bin/coveralls.js <coverage/lcov.info"
+    "build": "tsc",
+    "postbuild": "node build/postbuild.js",
+    "lint": "semistandard",
+    "lint:fix": "semistandard --fix",
+    "updateBinary": "ts-node build/cleanupbuild.js && ts-node ./downloadCurrentVersion.ts && ts-node ./verifyVersion.ts",
```

Why not `clean.js` or `distClean.js`?
```suggestion
    "updateBinary": "ts-node build/clean.js && ts-node ./downloadCurrentVersion.ts && ts-node ./verifyVersion.ts",
```

Also, why `ts-node`? It's not a .ts file.

And don't you need to rebuild before executing the .ts files?

---

> Why not `clean.js` or `distClean.js`?
> 
> Also, why `ts-node`? It's not a .ts file.
> 
> And don't you need to rebuild before executing the .ts files?

This has been updated to clean. 

And no, since these are only used during development, a development dependency `ts-node` is used which transpiles and runs the code in one hit. `ts-node` would of been used for testing `npm run test` but since during testing we do dynamic importing in test/compile.ts we cannot without a massive performance hit. 

The clean file was introduced to allow the download and verify to function correctly. If this is not done, the download and verify will attempt to look into /dist for the `soljson.js` file.  This ensures that no transpiled code exists and the download / verify can function on the new solcjson.js.